src/app/public/features/components/header/header.component.ts

```diff
@@ -3,13 +3,10 @@ import { Component, OnInit } from '@angular/core';
 @Component({
   selector: 'app-header',
   templateUrl: './header.component.html',
-  styleUrls: ['./header.component.css']
+  styleUrls: ['./header.component.css'],
 })
 export class HeaderComponent implements OnInit {
+  constructor() {}
 
-  constructor() { }
-
-  ngOnInit(): void {
-  }
-
+  ngOnInit(): void {}
```

Comment:
tip: empty NgOnInit should be avoided

Reply:
change added here in [WPA-button PR. ](https://github.com/MariaAlba1306/github/pull/4)